CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## v1.0.1
+
 ### Added
```

Comment:
since there's an added section, we shouldn't be releasing as a minor, let's use 1.1.0